examples/00-mapdl-examples/spotweld.py

```diff
@@ -0,0 +1,75 @@
+"""
+.. _ref_spotweld:
+
+=================================================
+Running an input file - spotweld SHELL181 example
+=================================================
+
+This MAPDL example demonstrates how to model spot welding on three
+thin sheets of metal. Here, the full input file is simply run using
+the PyMAPDL interface.
+
+Using the following commands, you can directly use an APDL
+script within a PyMAPDL session with the following
+commands:
+
+"""
+
+from ansys.mapdl.core import launch_mapdl
+
+mapdl = launch_mapdl()
+if mapdl._local:
```

Comment:
I would suggest to use the API: [mapdl.is_local](https://github.com/ansys/pymapdl/blob/fdf3a038bfdce4c7434625ea837db8cece8c7700/src/ansys/mapdl/core/mapdl.py#L4716)

Reply:
I will make that change, thanks!